docs/querying/operators.md

```diff
@@ -318,102 +318,167 @@ all other labels are preserved in the output. `by` does the opposite and drops
 labels that are not listed in the `by` clause, even if their label values are
 identical between all elements of the vector.
 
-`parameter` is only required for `topk`, `bottomk`, `limitk`, `limit_ratio`,
-`quantile`, and `count_values`. It is used as the value for _k_, _r_, φ, or the
-name of the additional label, respectively.
-
 ### Detailed explanations
 
-`sum` sums up sample values in the same way as the `+` binary operator does
-between two values. Similarly, `avg` divides the sum by the number of
-aggregated samples in the same way as the `/` binary operator. Therefore, all
-sample values aggregation into a single resulting vector element must either be
+#### `sum`
+
+`sum(v)` sums up sample values in `v` in the same way as the `+` binary operator does
+between two values. 
+
+All sample values aggregation into a single resulting vector element must either be
```

Comment:
```suggestion
All sample values being aggregated into a single resulting vector element must either be
```

(Not introduced by this PR, but needs fixing anyway.)

Reply:
Good catch, fixed in 95f7535.